nanomq_cli/include/nftp _client.h

```diff
@@ -0,0 +1,22 @@
+// Author: wangha <wangha at emqx dot io>
```

Comment:
Filename ????

Reply:
most codes are originally from wanghamax, but I will modify that. This PR is still wip, so chill. XD

Reply:
Fine. I have turned it to a draft pr